include/avif/avif.h

```diff
@@ -772,6 +795,16 @@ typedef struct avifDecoder
     // Set this via avifDecoderSetSource().
     avifDecoderSource requestedSource;
 
+    // If this is true and a progressive AVIF is decoded, avifDecoder will behave as if the AVIF
+    // is an image sequence, in that it will set imageCount to the available number of progressive
+    // frames available, and avifDecoderNextImage()/avifDecoderNthImage() will all for specific
+    // variants of a progressive image to be decoded. To distinguish between a progressive AVIF
```

Comment:
"will all for specific variants of a progressive image to be decoded" sounds strange. The word "all" seems incorrect, but I don't know what word it should be changed to.

Reply:
It should be `will allow for specific layers`. Fixing.